packages/rum/src/domain/record/startFullSnapshots.ts

```diff
@@ -65,13 +66,17 @@ export function startFullSnapshots(
     }
     return records
   }
-  fullSnapshotCallback(takeFullSnapshot())
+
+  fullSnapshotPendingCallback()
```

Comment:
**💬 suggestion:** ‏the snapshot is not pending here, you could remove this call.